app/forms/claim/mentors_form.rb

```diff
@@ -1,4 +1,5 @@
 class Claim::MentorsForm < ApplicationForm
+  include Rails.application.routes.url_helpers
```

Comment:
Personally, I would add this to `ApplicationForm`. Components, Views, Controller, etc, all have access to url helpers.